server.py

```diff
@@ -3,36 +3,49 @@
 from time import sleep
 from time import time
 class ClientChannel(PodSixNet.Channel.Channel):
-    def Network(self, data):
+    '''client channel class docstring to do'''
+
+    def Network(self, data: dict):  # Function name isn't very descriptive!
```

Comment:
Network is not a descriptive name. This function just prints the data object to the current stdout path, i.e. sys/stdout.
Seems to be purely a logging function.

Reply:
Yeah network is not a descriptive name, but I believe its automatically called whenever the server receives any message (following the convention of the other "Network_" functions).